apps/hobbies-helsinki/public/locales/en/cms.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
As far as I understand, the idea in the monorepo is to put these locales to common-i18n package

Reply:
https://github.com/belgattitude/nextjs-monorepo-example/tree/main/packages/common-i18n

Reply:
Could be. So we should move all our translations there? From every app (Hobbies, Events and common-event-components?)

Reply:
all translations moved. Deployed in #17